hibernate-core/src/test/java/org/hibernate/test/locking/PessimisticWriteLockTimeoutTest.java

```diff
@@ -79,6 +82,7 @@ public void testNoWait()
 
 	@Test
 	@RequiresDialect({ Oracle8iDialect.class, PostgreSQL95Dialect.class })
+	@RequiresDialectFeature(DialectChecks.SupportNoWait.class)
```

Comment:
I think `@RequiresDialectFeature` can be removed

Reply:
done